sapan-parikh/SAPAN_PARIKH.md

```diff
@@ -1,16 +1,34 @@
 # Not All Tests are the Same
+I have worked on projects on multiple occasions where I learned how all automated tests are not the same.
 
-I have gotten an opportunity to work on projects where I learned how automated testing is and that not all automated tests are the same. They differ from each other in at least two ways,speed of execution and risk of failure.
+They differ from each other in at least two ways, speed of execution and risk of failure.
 
-While the testing pyramid is QA 101, it'salso one of the most ignored learning. One project that I worked on depended heavily on unit testing, while the other relied heavily on automated() testing. approaches have their shortcomings.
+While the [testing pyramid](https://martinfowler.com/articles/practical-test-pyramid.html) is QA 101, it's also one of the most ignored learning. One project that I worked on depended heavily on unit testing, while the other relied heavily on Selenium-based UI testing. Both approaches have their shortcomings.
```

Comment:
I think this hyperlink text here should be "Test Pyramid" as the article heading in the link is also Test Pyramid as is the header for the section below